Getting-Started-with-InnerSource/jp.md

```diff
@@ -1,29 +1,29 @@
-# Chapter 1. A Robust Approach to Team Collaboration
+# チャプター1. チームコラボレーションへの確固たるアプローチ
 
-OS、クラウドコンピューティング、JavaScriptフレームワークなど、オープンソースソフトウェアの普及に刺激され、強力なオープンソースムーブメントの実践を真似て、_InnerSource_という名の社内コラボレーションを行う企業が続出しています。このページでは、インターネット商取引の大手PayPalの経験を読み、InnerSourceがエンジニア、経営者、マーケティング/PR部門にどのような利益をもたらすかを見ていきます。
+近年、オペレーティングシステム、クラウドコンピューティング、JavaScriptフレームワークをはじめとするさまざまなオープンソースソフトウェアの普及に触発され、_インナーソース_ という標語のもと、その強力なオープンソースのプラクティスを模倣して内部のコラボレーションに取り入れようとする企業が増えています。このページでは、インターネット商取引の大手である PayPal の経験から、インナーソースがエンジニア、経営者、マーケティング/PR部門にどのような利益をもたらすかを見ていきます。
 
-InnerSourceのプロジェクト管理の魅力を理解するために、何がそんなに成功したオープンソースのソフトウェア開発を作っていることを考えてみましょう。
+インナーソースによるプロジェクト管理の魅力を理解するために、そもそもオープンソースでのソフトウェア開発がなぜここまで成功してきたのか、その要因を考えてみましょう。
 
-* プログラマは、自分の仕事を、マネージャやチームだけでなく、広い範囲の人々と共有します。ほとんどのオープンソースプロジェクトでは、世界中の誰もが自由にコードを閲覧し、コメントし、コードを調べて新しいスキルを学び、コードを改善したり自分のニーズに合わせてカスタマイズしたりできると思う変更を提出することができます。
-* プロジェクトに基づく新しいコードリポジトリ（ブランチ）を自由に作成することができるので、コードの予期せぬ使い方をするサイトがコードを適応させることができる。異なるブランチを元のマスターブランチに再合成するためのルールや技術的なサポートが通常ある。
-* 地理的に離れた場所にいる人が、別々の時間に、同じコードで作業したり、同じプロジェクトに異なるコードのファイルを提供することができます。
-* コミュニケーションは、口コミで非公式に共有されるのではなく、文書化されて公開サイトに投稿される傾向があり、プロジェクトの歴史と新しいプロジェクトメンバーの学習の機会を提供します。
-* ユニットテストを書くことがプログラミングの重要な仕事になる。ユニットテストとは、ある条件下で不正な入力を拒否したり、適切な分岐を行うなど、特定の孤立した動作をチェックする小さなテストのことです。オープンソースやInnerSourceでは、本番稼動時の不具合から守るために、変更がチェックインされるたびにテストが行われます。
+* プログラマーは、マネージャーやチームだけでなく、幅広いオーディエンスと自分の仕事を共有します。ほとんどのオープンソースプロジェクトでは、世界中の誰もが自由にコードをみることができ、コメントを残し、検証することにより新しいスキルを学ぶことができ、また、ニーズに合わせてカスタマイズしたり、改善したコードの変更をサブミットすることができます。
+* 該当プロジェクトをもとにした新しいコードリポジトリ(ブランチ)を自由に作成できるため、想定外なコードの使用にも適応させることとができます。通常、さまざまなブランチを元のマスターブランチに再マージするためのルールと技術サポートがあります。
```

Comment:
原文からしてわかりづらいですね。
ブランチを分けられるから想定外な使用方法の実験を本流に影響させずにできます、という意味だと思うのですが、それが so that … can adapt it から明確には読み取れず、適応の意味も不明確になっているように見えます。私ならそれを解消するために以下のように訳します。

* …(ブランチ)を自由に作成できるため、コードの想定外な使い方を試したいというニーズにも応えることができます。

Reply:
一番わかりにくかったですね。ちょっと意訳的になってしまいました。
これに関してはネイティブの友人にもニュアンスをチェックしてもらったのですが、

> 「コードが元々に作られた目的以外の使用にも対応できます」のニュアンスが正確かな

というコメントをもらいました。

Reply:
理解できました。こういうときネイティブは頼りになりますね。